### ISSUE TITLE
(Installation Workflow) Package doesn't import in Blender 3.3.7 LTS

### ISSUE THREAD

**Issue by Vivraan** [cityjson/Up3date#22]
By default, the following the current installation instructions simply leads to a paste of the contents of this repo inside the user-storage addons folder, which is not ideal for organising addons. If you have cloned the repo elsewhere, it won't even copy all the contents, just the `__init__.py` script.

If one decides to clone the repository (which should be the best way to use this fantastic project), the ideal way is to do so directly in the addons folder location. For Windows users, it's `C:\Users\<USERNAME>\AppData\Roaming\Blender Foundation\Blender\<MAJOR_VERSION>.<MINOR_VERSION>\scripts\addons`.

**Comment by liberostelios** [comment]
Hey @Vivraan!

Thank you for your remark. I am not sure if I can follow you, though. The default installation instructions are to download a copy of the repository as a zip and install it through Blender's GUI which should copy everything. Is that what you followed and Blender didn't import anything except for `__init__.py`?

**Comment by Vivraan** [comment]
No, installing through a zip archive just pastes all the contents directly in the addons folder, which is not ideal.
